diff -rupN OLD/src/chuck_compile.cpp chuck-1.3.4.0/src/chuck_compile.cpp
--- OLD/src/chuck_compile.cpp	2014-10-09 10:04:49.907117341 -0400
+++ chuck-1.3.4.0/src/chuck_compile.cpp	2014-10-10 11:46:23.597242569 -0400
@@ -46,6 +46,7 @@
 #include "ulib_opsc.h"
 #include "ulib_regex.h"
 #include "chuck_io.h"
+#include "blackbox/blackbox.h"
 
 #if defined(__PLATFORM_WIN32__)
 #include "dirent_win32.h"
@@ -590,7 +591,9 @@ t_CKBOOL load_internal_modules( Chuck_Co
     EM_log( CK_LOG_SEVERE, "class 'RegEx'..." );
     if( !load_module( env, regex_query, "RegEx", "global" ) ) goto error;
     // if( !load_module( env, net_query, "net", "global" ) ) goto error;
-    
+   
+    #include "blackbox/modules.cpp"
+ 
     if( !init_class_HID( env ) ) goto error;
     if( !init_class_serialio( env ) ) goto error;
         
diff -rupN OLD/src/makefile chuck-1.3.4.0/src/makefile
--- OLD/src/makefile	2014-10-09 10:04:50.199118796 -0400
+++ chuck-1.3.4.0/src/makefile	2014-10-10 11:45:50.025076084 -0400
@@ -75,6 +75,8 @@ ifneq (,$(strip $(filter osx-rl,$(MAKECM
 include makefile.rl
 endif
 
+include blackbox/makefile
+
 CSRCS+= chuck.tab.c chuck.yy.c util_math.c util_network.c util_raw.c \
 	util_xforms.c
 CXXSRCS+= chuck_absyn.cpp chuck_parse.cpp chuck_errmsg.cpp \
